examples/amazon-pay-button-example/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "eslint.experimental.useFlatConfig": true
```

Comment:
why is it here? I do not see any eslint flat config

Reply:
was part of nuxt module starter boilerplate, I removed it